packages/jsii-pacmak/lib/targets/dotnet/dotnetdocgenerator.ts

```diff
@@ -167,8 +165,10 @@ export class DotNetDocGenerator {
       example,
       TargetLanguage.CSHARP,
       enforcesStrictMode(this.assembly),
+      undefined,
+      true /* prefix disclaimer */,
```

Comment:
This argument should be an option to the `Rosetta` object, not to the individual calls to `translateSnippet`. It's an "operating mode" of the translator.